<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Luisa Bermeo</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COVID-19 Data Visualization (Data source: Johns Hopkins University)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpControls">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>761</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Controles</string>
    </property>
    <widget class="QSlider" name="sldAverageDays">
     <property name="geometry">
      <rect>
       <x>549</x>
       <y>40</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Average
# of days</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="pnlOptions">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>79</y>
       <width>120</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Option</string>
     </property>
     <widget class="QRadioButton" name="rdbCumulative">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cumulative</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdbDaily">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Daily</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="pnlDataPlot">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>120</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Data to plot</string>
     </property>
     <widget class="QRadioButton" name="rdbCases">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cases</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdbDeaths">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Deaths</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdbBoth">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Both</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="pnlCuntry">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>211</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>Country</string>
     </property>
     <widget class="QListView" name="listCountries">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="pnlState">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>211</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>State or Region</string>
     </property>
     <widget class="QListView" name="listStates">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="pnlPlot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="10"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
